### ISSUE TITLE
[Feature Request] Tilepicker

### ISSUE THREAD

**Issue by Mix3rn** [deepnight/ldtk#845]
Hi.
It would be really nice to have an option to disable the Tilepicker-auto-widening-system when hovering the mouse over the window in the editor.

**Comment by coavins** [comment]
You should probably improve the title of this issue, it sounds like you're asking for a tilepicker.

**Comment by zissakos** [comment]
I had the exact same issue some weeks ago: #840 
It's a little annoying, especially if the tileset already fits into the sidepanel and there's no need for it to grow on hover.
Can this please either made configurable or a little smarter (only pop up if tileset doesn't fit)?

**Comment by Mix3rn** [comment]
Exactly, should be a configurable setting because you can zoom in/out of the tilepicker-window anyway
